### PR TITLE
change delay metric plot 

### LINKED ISSUE #1082: Change plot for delay metric
The delay metric looks good, I've merged it into master, but I'd like the plot that is currently put under ITERS to instead be at the top level directory.

Name it "delayTotalByLinkCapacity" and it should be a bar chart with one bar per iteration and it should be colored according to the capacity bins (each bin a different color).

You can remove the 0.delayTotalByLinkCapacity.png plot under ITERS as it will be totally redundant with this one.

I've already started a branch on this (to adjust the capacity bins), you can work from this:

cs/#1082-refine-analysis

### PR DESCRIPTION
**Pull request by rajnikantsh**
Fixed #1082 

Change the delayTotalByLinkCapacity plot .

---
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/1083)

### CODE REVIEW

**Comment by colinsheppard** [comment]
@rajnikantsh something isn't right. the 2 delay plots in the base directory no look like this:

delaytotalbylinkcapacity

and this

delayaverageperkilometer

**Comment by colinsheppard** [comment]
the delay per kilometer plot shouldn't be split by color, should just be a single bar.

also, can you change the y-axis on the plot "delayTotalByLinkCapacity" to read "Total Delay (hour)".

Thank you.

File: src/main/scala/beam/analysis/DelayMetricAnalysis.scala [hunk 1]
```diff
@@ -142,29 +141,33 @@ class DelayMetricAnalysis @Inject()(
...
   }
 
-  def createDelayCapacityGraph(dataset: CategoryDataset, iterationNumber: Int, fileName: String): Unit = {
+  def createDelayCapacityGraph(fileName: String): Unit = {
     val chart = GraphUtils.createStackedBarChartWithDefaultSettings(
-      dataset,
+      delayTotalByLinkCapacityDataset,
       graphTitle,
-      xAxisName,
+      "iteration",
```
[the remark this answers is not in the document]
Comment (rajnikantsh) [reply src/main/scala/beam/analysis/DelayMetricAnalysis.scala]: I have changed the xAxis as per your comment.

**Comment by rajnikantsh** [comment]
> the delay per kilometer plot shouldn't be split by color, should just be a single bar.
> 
> also, can you change the y-axis on the plot "delayTotalByLinkCapacity" to read "Total Delay (hour)".
> 
> Thank you.

Hi @colinsheppard, I have resolved this issue and pushed the changes.

**Pull request merged**